graph/index.js

```diff
@@ -0,0 +1,33 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+const ethers = require('ethers');
+const sapphire = require('@oasisprotocol/sapphire-paratime');
+
+const app = express();
+app.use(bodyParser.json());
+
+const provider = new ethers.providers.JsonRpcProvider(process.env.RPC);
+const signer = new ethers.Wallet(Buffer.from(ethers.utils.randomBytes(32)).toString('hex'), provider);
```

Comment:
```suggestion
const signer = ethers.Wallet.createRandom();
```

Reply:
Need to connect signer to JsonRpcProvider then I guess

Reply:
Sorry, I meant `ethers.Wallet.createRandom().connect(...)`. In fact, you don't really need a signer. You can set the `from` address to `address(0)`. I think that'll work in this case, actually.